src/server/config.ts

```diff
@@ -127,6 +127,7 @@ export interface SessionSettings {
 
 export const validEmailDomainGlobExpression = process.env
   .VALID_EMAIL_GLOB_EXPRESSION as string
+export const domainPattern = `*@${validEmailDomainGlobExpression}`
```

Comment:
- this really is an e-mail address pattern, not a domain pattern. rename accordingly.
- it's a bit concerning that we have two validators both for validating e-mails. could we figure out which one we want to use?

Reply:
not to mention the security implications if we overcomplicate things here